vendor/vendor.json

```diff
@@ -0,0 +1,268 @@
+{
+	"comment": "",
+	"ignore": "test",
+	"package": [
+		{
+			"checksumSHA1": "E2a5TaCbwqs7PwP1fvfWLhvgr38=",
+			"comment": "v2.2.0-rc.0-72-g106d918",
+			"path": "github.com/coreos/etcd/Godeps/_workspace/src/github.com/coreos/go-systemd/journal",
```

Comment:
I don't see this path in the old Godeps. why does it show for govendor? did you vendor a snapshot of the entire repo, or only the referenced packages? (same comment applies for several subpackages)